src/OSPSuite.FuncParserNative/include/FuncParser/Constant.h

```diff
@@ -14,7 +14,7 @@ class Constant
 		double _value;
 	
 	public:
-		Constant (const std::string & Name, double Value);
+		explicit Constant (const std::string & Name, double Value);
```

Comment:
I don't see a point in defining explicit constructors with **multiple** arguments